### DISCUSSION TITLE
Can I sync files from another repository to the current one

### DISCUSSION THREAD

**Discussion by gfb-47** [BetaHuhn/repo-file-sync-action#158] [Q&A] [answered]
I'm having a failure because I'm trying to get files from an open source repo (which I don't have access), any solution to this?

**Comment by BetaHuhn** [comment, marked as answer]
You mean you want to sync files from a public repository not owned by you to a private repository you own?

**Reply by gfb-47** [reply]
Yes!! I found another action for that: [gh_fsync](https://github.com/marketplace/actions/gh_fsync)

**Reply by BetaHuhn** [reply]
Ah okay, yeah that isn't possible with this action. You specify files to sync from the repository the action is running in ***to*** a external repository. `gh_fsync` is the other way around, you specify ***from*** where to sync external files to your repository (what you were looking to do).

I hope this explanation clears things up.

**Reply by gfb-47** [reply]
Got it! Thanks
